dts/arm/nxp/nxp_imx94x.dtsi

```diff
@@ -752,221 +784,221 @@
+		 <&iomuxc_gpio_io56_gpio_io_gpio3_io24>,
+		 <&iomuxc_gpio_io57_gpio_io_gpio3_io25>;
 };
 
-&gpio4{
+&gpio4 {
 	pinmux = <&iomuxc_ccm_clko1_gpio_io_gpio4_io0>,
-		<&iomuxc_ccm_clko2_gpio_io_gpio4_io1>,
-		<&iomuxc_ccm_clko3_gpio_io_gpio4_io2>,
-		<&iomuxc_ccm_clko4_gpio_io_gpio4_io3>,
-		<&iomuxc_dap_tdi_gpio_io_gpio4_io4>,
-		<&iomuxc_dap_tms_swdio_gpio_io_gpio4_io5>,
-		<&iomuxc_dap_tclk_swclk_gpio_io_gpio4_io6>,
-		<&iomuxc_dap_tdo_traceswo_gpio_io_gpio4_io7>,
-		<&iomuxc_sd1_clk_gpio_io_gpio4_io8>,
-		<&iomuxc_sd1_cmd_gpio_io_gpio4_io9>,
-		<&iomuxc_sd1_data0_gpio_io_gpio4_io10>,
-		<&iomuxc_sd1_data1_gpio_io_gpio4_io11>,
-		<&iomuxc_sd1_data2_gpio_io_gpio4_io12>,
-		<&iomuxc_sd1_data3_gpio_io_gpio4_io13>,
-		<&iomuxc_sd1_data4_gpio_io_gpio4_io14>,
-		<&iomuxc_sd1_data5_gpio_io_gpio4_io15>,
-		<&iomuxc_sd1_data6_gpio_io_gpio4_io16>,
-		<&iomuxc_sd1_data7_gpio_io_gpio4_io17>,
-		<&iomuxc_sd1_strobe_gpio_io_gpio4_io18>,
-		<&iomuxc_sd2_vselect_gpio_io_gpio4_io19>,
-		<&iomuxc_sd2_cd_b_gpio_io_gpio4_io20>,
-		<&iomuxc_sd2_clk_gpio_io_gpio4_io21>,
-		<&iomuxc_sd2_cmd_gpio_io_gpio4_io22>,
-		<&iomuxc_sd2_data0_gpio_io_gpio4_io23>,
-		<&iomuxc_sd2_data1_gpio_io_gpio4_io24>,
-		<&iomuxc_sd2_data2_gpio_io_gpio4_io25>,
-		<&iomuxc_sd2_data3_gpio_io_gpio4_io26>,
-		<&iomuxc_sd2_reset_b_gpio_io_gpio4_io27>,
-		<&iomuxc_sd2_gpio0_gpio_io_gpio4_io28>,
-		<&iomuxc_sd2_gpio1_gpio_io_gpio4_io29>,
-		<&iomuxc_sd2_gpio2_gpio_io_gpio4_io30>,
-		<&iomuxc_sd2_gpio3_gpio_io_gpio4_io31>;
+		 <&iomuxc_ccm_clko2_gpio_io_gpio4_io1>,
+		 <&iomuxc_ccm_clko3_gpio_io_gpio4_io2>,
+		 <&iomuxc_ccm_clko4_gpio_io_gpio4_io3>,
+		 <&iomuxc_dap_tdi_gpio_io_gpio4_io4>,
+		 <&iomuxc_dap_tms_swdio_gpio_io_gpio4_io5>,
+		 <&iomuxc_dap_tclk_swclk_gpio_io_gpio4_io6>,
+		 <&iomuxc_dap_tdo_traceswo_gpio_io_gpio4_io7>,
+		 <&iomuxc_sd1_clk_gpio_io_gpio4_io8>,
+		 <&iomuxc_sd1_cmd_gpio_io_gpio4_io9>,
+		 <&iomuxc_sd1_data0_gpio_io_gpio4_io10>,
+		 <&iomuxc_sd1_data1_gpio_io_gpio4_io11>,
+		 <&iomuxc_sd1_data2_gpio_io_gpio4_io12>,
+		 <&iomuxc_sd1_data3_gpio_io_gpio4_io13>,
+		 <&iomuxc_sd1_data4_gpio_io_gpio4_io14>,
+		 <&iomuxc_sd1_data5_gpio_io_gpio4_io15>,
+		 <&iomuxc_sd1_data6_gpio_io_gpio4_io16>,
+		 <&iomuxc_sd1_data7_gpio_io_gpio4_io17>,
+		 <&iomuxc_sd1_strobe_gpio_io_gpio4_io18>,
+		 <&iomuxc_sd2_vselect_gpio_io_gpio4_io19>,
+		 <&iomuxc_sd2_cd_b_gpio_io_gpio4_io20>,
+		 <&iomuxc_sd2_clk_gpio_io_gpio4_io21>,
+		 <&iomuxc_sd2_cmd_gpio_io_gpio4_io22>,
+		 <&iomuxc_sd2_data0_gpio_io_gpio4_io23>,
+		 <&iomuxc_sd2_data1_gpio_io_gpio4_io24>,
+		 <&iomuxc_sd2_data2_gpio_io_gpio4_io25>,
+		 <&iomuxc_sd2_data3_gpio_io_gpio4_io26>,
+		 <&iomuxc_sd2_reset_b_gpio_io_gpio4_io27>,
+		 <&iomuxc_sd2_gpio0_gpio_io_gpio4_io28>,
+		 <&iomuxc_sd2_gpio1_gpio_io_gpio4_io29>,
+		 <&iomuxc_sd2_gpio2_gpio_io_gpio4_io30>,
+		 <&iomuxc_sd2_gpio3_gpio_io_gpio4_io31>;
 };
 
-&gpio5{
+&gpio5 {
 	pinmux = <&iomuxc_eth0_txd0_gpio_io_gpio5_io0>,
-		<&iomuxc_eth0_txd1_gpio_io_gpio5_io1>,
```

Comment:
Please don't randomly fix whitespaces in the middle of a patch. You can create a separate cleanup PR if you want .

Reply:
> Please don't randomly fix whitespaces in the middle of a patch. You can create a separate cleanup PR if you want .

Fix compliance checks issues.